### ISSUE TITLE
Identify issues  to be added or removed from A11y statement

### ISSUE THREAD

**Issue by YedidaZalik** [cds-snc/notification-planning#2415]
Sources to review for due dligence in changing A11y statement
- [x] Yedida to check all items marked in [table column for A11y statement ](https://github.com/orgs/cds-snc/projects/68/views/1?sortedBy%5Bdirection%5D=desc&sortedBy%5BcolumnId%5D=218625312&sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=Title)whether to add or remove from statement
- [x]  Phil to review Yedida's list of items not selected for statement to confirm nothing there that should be in statement
- [x]  Phil and Yedida to review shortlist of items for statement along with other document [Feedback for Accessibility Statement 2025 (for Yedida)](https://docs.google.com/document/d/19RbzKCuE2i51uGUGBZJKzDtJfcG4dVjAZYqX8uxsHGg/edit?tab=t.0#heading=h.ed4n6id2933o)
- [x]  Identify resolved issues that can be deleted from statement and confirm with devs/Phil and document in case we're ever asked about these
- [x]  Identify issues that need to be added to statement

**Comment by YedidaZalik** [comment]
Reviewed "No status" column with Phil and actioned some items
Several items identified for statement
Phil to review "Fixed" "Not an issue" "Part fixed/part not issue" to confirm correct labels
**To ask at stand-up:** Plan for outstanding issues?

**Comment by schancase** [comment]
Q in standup - where should we document the issues we are determining to remove from the A11y statement? To discuss async

**Comment by YedidaZalik** [comment]
We finished this and documented in https://docs.google.com/spreadsheets/d/1V2gqBBUkODQFoGfqSDUQe5cpMe5xBWCOKFLwzhBkiwc/edit?gid=0#gid=0
